src/Console/Application.php

```diff
@@ -130,4 +144,76 @@ private function createContext(InputInterface $input, OutputInterface $output):
 
         return $descriptor->function->invoke(...$args);
     }
+
+    private function displayUpdateWarningIfNeeded(SymfonyStyle $symfonyStyle): void
+    {
+        $latestVersion = $this->cache->get('latest-version', function (ItemInterface $item): array {
+            $item->expiresAfter(3600 * 60);
+            $opts = [
+                'http' => [
+                    'method' => 'GET',
+                    'header' => 'User-Agent: castor',
+                ],
+            ];
+
+            $context = stream_context_create($opts);
+
+            $content = @file_get_contents('https://api.github.com/repos/jolicode/castor/releases/latest', false, $context);
```

Comment:
Can you add a timeout on this? (maybe it's time to require symfony/http-client

Reply:
I did it on the stream context